src/main/scala/io/iohk/ethereum/network/EtcPeerManagerActor.scala

```diff
@@ -112,7 +118,7 @@ class EtcPeerManagerActor(peerManagerActor: ActorRef, peerEventBusActor: ActorRe
     * @return new updated peer info
     */
   private def handleSentMessage(message: Message, initialPeerWithInfo: PeerWithInfo): PeerInfo =
```

Comment:
Do we need `message: Message`?

Reply:
we do not, but `updatePeersWithInfo` expect function with shape ` messageHandler: (Message, PeerWithInfo) => PeerInfo` that why i would leave it as it is.